src/main/java/com/google/sps/servlets/Util.java

```diff
@@ -171,15 +172,29 @@ public void associateRefreshToken(String userId, String refreshToken) {
     datastore.put(tokenEntity);
   }
 
-  public void updateDevices(String userId, List<String> deviceIds, String updatesInJson) throws IOException {
+  public List<String> updateDevices(String userId, List<String> deviceIds, String updatesInJson) throws IOException {
```

Comment:
consider adding a comment to the function. The function name says `updateDevices` but returns a `List<String>`. People may wonder what's returned without dive deep into the function.

Reply:
I am planning on making a code health PR soon that will have additional comments/javadocs for functions/fields